src/java/org/apache/nutch/storage/Duplicate.java

```diff
@@ -0,0 +1,271 @@
+/**
```

Comment:
Please add the duplicate.avsc file from which this file is compiled. Thanks

Reply:
I made a duplicate.avsc file. I actually hadn't known that these storage types were compiled from .avsc files. What is the process by which they are compiled?

Reply:
The ```generate-gora-src``` target achieves this

https://github.com/apache/nutch/blob/2.x/build.xml#L616-L627

Reply:
Thanks, should I add an argument to that then like
```xml
<arg value="src/gora/duplicate.avsc"/>
```

Reply:
Yes

Reply:
You should then run the target.